pydjamodb/test_runner.py

```diff
@@ -4,17 +4,16 @@
 from django.db import connections
 from django.test.runner import ParallelTestSuite, DiscoverRunner
 
+from germanium.signals import set_up, tear_down
+
+from .connection import TestTableConnection
 from .models import dynamodb_model_classes
 
 
 def init_pynamodb_test_prefix(prefix=None):
     for model_class in dynamodb_model_classes:
         model_class._connection = None
```

Comment:
for other reviewers, this line is necessary for the `model_class._get_connection()`